_storage/overview.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: storage
+permalink: /:collection/:path.html
+---
+# A decentralized storage architecture
+
+The Blockstack Network stores application data using a storage service called
+Gaia. Transactional metadata is stored on the Blockstack blockchain and user
+application data is stored in Gaia hub. Storing data off of the blockchain
+ensures that Blockstack applications can provide users with high-performance and
+high availability for data reads and writes without introducing central trust
+parties.
+
+* TOC
+{:toc}
+
+##  Understand Gaia in the Blockstack architecture
+
+The following diagram depicts the Blockstack architecture and Gaia's place in it:
+
+{% include architecture.md %}
+
+## User control or how is Gaia decentralized?
+
+Within the provider's storage system a Gaia hub runs as a service. This service
+authenticates writes to a location by requiring a valid authentication token from a requestor.
+
+![Gai storage](/storage/images/gaia-storage.png)
+
+Gaia's approach to decentralization focuses on user control of data and storage.
+If a user can choose which storage provider to store data with, then that is all the decentralization required to enable user-controlled
+applications.
+
+The control of user data lies in the way that user data is accessed.
+When an application fetches a file `data.txt` for a given user `alice.id`, the
+lookup will follow these steps:
+
+1. Fetch the `zonefile` for `alice.id`
+2. Read her profile URL from her `zonefile`.
+3. Fetch Alice's profile.
+4.  _Verify_ that the profile is signed by `alice.id`'s key
+5. Read the application root URL (e.g. `https://gaia.alice.org/`) out of the profile
+6. Fetch file from `https://gaia.alice.org/data.txt`.
+
+Because `alice.id` controls her zonefile, she can change where her profile is
```

Comment:
"Because `alice.id` controls her zonefile **with her Blockstack ID**"?

Reply:
`alice.id` = the Blockstack ID

`Because alice.id controls her zone file with alice.id` doesn't actually make sense. 

Alice controls her zonefile by virtue of the fact that she has the secret key for it.

`Because `alice.id` has access to her zonefile, she can change where her profile
is stored. `

Remember, we don't actually have a migration yet. So, this is a bit of hand waving.